birthdays-project/config/settings/dev.py

```diff
@@ -3,3 +3,15 @@
 # Enable debug
 DEBUG = True
 TEMPLATES[0]['OPTIONS']['debug'] = DEBUG
+
+DATABASES = {
```

Comment:
Remove the `DATABASES` dictionary. It will override the postgresql config from base.py